eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -125,5 +125,3 @@ stages:
         ArtifactName: packages
         TargetDocRepoOwner: ${{ parameters.TargetDocRepoOwner }}
         TargetDocRepoName: ${{ parameters.TargetDocRepoName }}
-        ${{ if eq(parameters.ReleaseDependsOnLiveTests, 'false') }}:
```

Comment:
@pallavit @jsquire I just wanted to let you know I'm removing this feature that @hallipr added about a year ago which forced folks that disabled live tests for release to ignore the failures. This feature hasn't been used even once in the last year and given our loosing up on release approvers I'm removing this also. Please let me know if you have any concerns.

Reply:
No objections from me.